server/container/guice/pom.xml

```diff
@@ -74,6 +75,20 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>${james.groupId}</groupId>
+                <artifactId>james-server-cassandra-ldap-guice</artifactId>
+                <version>${project.version}</version>
+                <type>test-jar</type>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>${james.groupId}</groupId>
+                <artifactId>james-server-data-ldap-integration-testing</artifactId>
```

Comment:
This dependency is not related to guice. We should add it in `james-project/pom.xml` **dependencyManagement** section.